### DISCUSSION TITLE
Welcome to startupMAXIM Discussions!

### DISCUSSION THREAD

**Discussion by mhmAdmin1122** [mhmAdmin1122/startupMAXIM#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hasaanmumtaz6** [comment]
So know what you think about footer design after header design

**Reply by mhmAdmin1122** [reply]
should you help me to design it

**Reply by hasaanmumtaz6** [reply]
why not check out this design spority.vercel.app
